packages/logs-async-reporter/logs-async-reporter.1.3/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "Logs reporter compatible with Async"
+description: "Logs reporter that will play nice with Async's runtime."
+maintainer: ["Vincent Bernardoff <vb@luminar.eu.org>"]
+authors: ["Vincent Bernardoff <vb@luminar.eu.org>"]
+license: "ISC"
+tags: ["async" "logs"]
+homepage: "https://github.com/vbmithr/logs-async"
+doc: "https://vbmithr.github.io/logs-async"
+bug-reports: "https://github.com/vbmithr/logs-async/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.10"}
+  "logs" {>= "0.7.0"}
+  "fmt" {>= "0.9.0"}
+  "core" {>= "0.16"}
+  "async" {>= "0.16"}
+  "zstandard" {>= "0.16"}
```

Comment:
```suggestion
  "core" {>= "v0.16"}
  "async" {>= "v0.16"}
  "zstandard" {>= "v0.16"}
```
same as #24559, it seems that the lower bounds spec for Jane Street packages are incorrect.